### PR TITLE
Feature/append query string parameters

### LINKED ISSUE #1965: Add 'Append Query String Parameters' field to Proxy Backend form
# User story
> As an **API Manager**
I want to **add query parameters to every API request** made from the proxy
so that end users do not need to see or add the query parameters themselves

# Context
 API Umbrella, or primary proxy, has a field called 'Append Query String Parameters' on the Backend configuration form. However, we have not 'mirrored' the 'query string parameters' field in our user interface.

## API Umbrella screenshot
API Umbrella showing query parameters and request headers fields

# Feature
Add the 'query string parameters' field to our schema and user interface.

# Original idea
To make calling of prefix simple even with queryParams 

https://apinf.io/myPrefix -> https://proxyproxyservicesite.com/query?queryParam1=http://www.finalproxytarget.site/catalog/subcatalog/show.php&queryParam2=find('%20.my-class%20div')&queryParam3=true

Instead of (mapping myPrefix to query)

https://apinf.io/myPrefix?queryParam1=http://www.finalproxytarget.site/catalog/subcatalog/show.php&queryParam2=find('%20.my-class%20div')&queryParam3=true

(Tried - perhaps it is older version of apinf - in APIKA -> https://umbrella.digipalvelutehdas.fi/urheilukellot/?api_key=YOUR_KEY&site=gigantti.fi/catalog/puhelimet-ja-gps/fi-urheilukellot/urheilukellot&selector=find(%27a.product-name%27)&forceText=true )

And leaving for the user's responsibility all the queryparams.

### PR DESCRIPTION
**Pull request by nnn**
Closes #1965

### CODE REVIEW

File: proxy_backends/client/form/form.less [hunk 1]
```diff
@@ -3,15 +3,12 @@
   padding-left: 0;
 
   .form-group {
-    margin-bottom: 0;
-
     .checkbox {
-      margin-bottom: 0.3em;
+      margin-bottom: 0em;
```
Comment (brylie) [inline proxy_backends/client/form/form.less]: When using zero, you don't need to specify any units.

File: proxy_backends/collection/apiUmbrellaSchema.js [hunk 2]
```diff
@@ -48,6 +48,19 @@ const SettingsSchema = new SimpleSchema({
     type: [RateLimitSchema],
     optional: true,
   },
+  append_query_string: {
+    type: String,
+    optional: true,
+    label: "Append Query String Parameters",
```
Comment (brylie) [inline proxy_backends/collection/apiUmbrellaSchema.js]: Remember i18n.

File: proxy_backends/collection/apiUmbrellaSchema.js [hunk 3]
```diff
@@ -48,6 +48,19 @@ const SettingsSchema = new SimpleSchema({
...
+  },
+  headers_string: {
+    type: String,
+    autoform: {
+      rows: 3,
+    },
+    optional: true,
+    label: "Set Request Headers",
```
Comment (brylie) [inline proxy_backends/collection/apiUmbrellaSchema.js]: Remember i18n.

**Comment by nnn** [comment]
@matleppa @jawidahmadi @brylie any other comments/suggestions or it can be merged?

**Pull request merged**